language/move-prover/tests/sources/defines.move

```diff
@@ -16,4 +16,27 @@ module TestDefines {
         aborts_if !in_range(x + y, 0, 18446744073709551615);
         ensures eq(result, x + y);
     }
+
+    // Spec functions accessing global state.
```

Comment:
A very minor comment. Just to be consistent, other test files are using two `// -----------------` for section headers.

Reply:
Done.